Add words from our new global dictionary.

geeknoid opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/istio/mixer/809)
<!-- Reviewable:end -->

istio-testing commented:
Jenkins job mixer/presubmit passed

rkpagadala commented on pkg/api/grpcServer.go:
```diff
@@ -61,9 +61,119 @@ type (
 var (
-	// TODO: need full list
+	// global word list as defined in https://github.com/istio/api/blob/master/mixer/v1/global_dictionary.yaml
 	globalWordList = []string{
```
why is globalWordList a var and not a const?

ZackButcher replied:
You can't declare const arrays in golang, unfortunately.

rkpagadala replied:
Ahh.. Thanks Zack

istio-testing commented:
Jenkins job mixer/pilot-regression passed

istio-testing commented:
Jenkins job mixer/e2e-smoketest passed

qiwzhang commented on pkg/api/grpcServer.go:
```diff
@@ -61,9 +61,119 @@ type (
 
 var (
-	// TODO: need full list
+	// global word list as defined in https://github.com/istio/api/blob/master/mixer/v1/global_dictionary.yaml
```
add comment to state which version of the file. now it is the first version.
we can add a version comment in the global dictionary file next time we update it.

geeknoid replied:
The URL now includes the specified commit SHA.

douglas-reid commented on pkg/api/grpcServer.go:
```diff
@@ -61,9 +61,119 @@ type (
 )
 
 var (
```
would it be worth having this in a separate file -- something like //pkg/api/globalwordlist.go ?  It would be cool if we could autogen from the global_dictonary.yaml too.

ZackButcher replied:
Is there a compelling reason to hard code this over ingesting it as a yaml file? Then we can just use the global_dictonary.yaml directly. Eventually we'll need to manage rolling out changes to this list (and probably handle having multiple versions to serve with at the same time), so we'll want it to not be compiled in anyway.

geeknoid replied:
https://docs.google.com/document/d/1NcGgEzSEt0A0_qqJfA3x6ETNyomyMrhm1sBlpdorbHY/edit#heading=h.la7bf3j6tcb5 explains the update procedures for the dictionary. The reason we want to build this into the code is that it's not really config. I don't want operators to mess around with this stuff willy nilly.

geeknoid commented:
I added a reference to the specific commit the list comes from, and I split the word list into its file such that we can potentially code gen the thing directly from the yaml definition file.

PTAL.

istio-testing commented:
Jenkins job mixer/presubmit passed
